homeassistant/components/tomorrowio/__init__.py

```diff
@@ -299,6 +299,15 @@ async def _async_update_data(self) -> dict[str, Any]:
                         TMRW_ATTR_WIND_GUST,
                     ],
                     [
+                        TMRW_ATTR_CLOUD_COVER,
+                        TMRW_ATTR_DEW_POINT,
+                        TMRW_ATTR_FEELS_LIKE,
+                        TMRW_ATTR_HUMIDITY,
```

Comment:
Duplicated

Reply:
I guess that was due to a misunderstanding on how that function works exactly. I thought the first list is for (realtime) sensors, and the second for forecasts. Are they all equally available? (Why the differentiation then?)

Reply:
That's correct, but you have duplicates in this list.

Reply:
Also, move the added items to the bottom instead of adding at the top.